index.html

```diff
@@ -2128,6 +2256,213 @@ <h4>Bidirectional text</h4>
+        a direct child of the <code>&lt;body&gt;</code> element.
+        That <code>&lt;div&gt;</code> element could be nested in another <code>&lt;div&gt;</code> element.
+        Therefore, it is possible to serialize the objects and properties of a <a>DAPT Script</a> into various <a>DAPT Documents</a>.
+        This section defines how to interoperably and unambiguously reconstruct a DAPT model from a <a>DAPT Document</a>.</p>
+      <p class="note">DAPT does not define a complete serialization of the DAPT data model for extensibility reasons,
+        to allow future versions to do so if needed.
+        Additionally, a <a>DAPT Document</a> can contain elements or attributes that
+        are not mentioned in the representations of DAPT objects or properties.
+        This could be because it has been generated by a processor conformant to some future version of DAPT,
+        or through a generic [[TTML2]] process,
+        or because it uses optional features, for example to add styling or layout.
+        This section defines how to process those elements or attributes.</p>
+      <p class="note">It is also possible to process <a>DAPT Documents</a> using generic [[TTML2]] processors,
+        which do not necessarily map the documents to the DAPT data model.
+        For example a generic TTML2 <a>presentation processor</a> could render an audio mix
+        based on a DAPT document without needing to model Script Events <i>per se</i>.
+        In that case, this section can be ignored.</p>
+      <section class="informative">
+        <h3>Early identification of non-conformant documents</h3>
+        <p><em>Normative provisions relating to this section are defined in [[TTML2]].</em></p>
+        <p>Since it is a requirement of DAPT that <a>DAPT Documents</a> include a
+        <code>ttp:contentProfiles</code> attribute on the root element,
+        and that the attribute includes a DAPT <a>content profile</a> designator,
+        as specified at <a href="#ttp-contentprofiles"></a>,
+        it follows that any TTML document that does not include such an attribute,
+        or does not include such a profile designator,
+        can be considered not to be a <a>DAPT Document</a>;
+        therefore a processor requiring strict adherence to DAPT could stop processing
+        such a document.</p>
+      </section>
+      <section class="informative">
+        <h3>Avoiding unverified profile conformance signalling</h3>
+        <p>If a processor takes as its input a <a>DAPT document</a> that claims conformance to
+          one or more content profiles which that processor does not support,
+          allows the document to be modified, and then outputs that modified document,
+          <a href="#ttp-contentprofiles"></a> prohibits it from including in <code>ttp:contentProfiles</code>
+          the values of those content profiles which that processor does not support.
+        </p>
+      </section>
```

Comment:
I still think that section is not needed. It is already captured in "5.6.2 ttp:contentProfiles". I would delete the entire section.

Reply:
Agreed, done in f69d320.